lcls-twincat-motion/Library/POUs/Motion/States/FB_StatesInputHandler.TcPOU

```diff
@@ -84,6 +86,10 @@ CASE nState OF
         END_IF
 END_CASE
 
+IF bMoveError THEN
+    bExecMove := FALSE;
```

Comment:
Ah, I think this is the line that explains my above question...

Reply:
This is indeed why it works out, but I think your previous point still stands